_templates/how-to.md

```diff
@@ -1,26 +1,72 @@
 ---
 title: [Character Limit: 255, Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. The article title should be a combination of the main problem statement and if applicable, the key environment statement. Also determines the title in the table of contents.]
-description: [Brief description of the How-To article]
-type: [HowTo]
-page_title: [Text that will populate the HTML file’s head:title attribute]
+description: [Character Limit: 255, Recommendation is to have <= 155 characters for a description as this can impact indexing by search engines like Google, Bing, etc. The description should be brief overview of the how-to article. It should exclude unnecessary phrases like "This article contains ..."]
+type: [how-to]
+page_title: [Text that will populate the HTML file’s head:title attribute. Normally, this should be the same as the title field.]
```

Comment:
About the "Normally..." part - this is not exactly right. Usually, it might be a good idea to provide different title and page title because this way you can include more keywords which will be hit by search engines. In addition, the page_title might contain "service" information such as the site title. For example "Cool Article | Knowledge Base | Telerik Platform" or "Cool Article | Telerik Platform".